.editorconfig

```diff
@@ -0,0 +1,12 @@
+[*]
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+[{*.py,*.cfg}]
+indent_size = 4
+
+[Makefile]
+indent_style = tab
```

Comment:
Refers to the .editorconf itself or the Makefile?

.editorconf is a standard to enforce simple code-conventions like line indentations and trailing white spaces. Good Editors support it. Team / Project conventions are meet more easly and less warnings from check tools.